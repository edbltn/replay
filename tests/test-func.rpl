int[] a = int[6];
a[5] = 27;

int f(int[] a) {
  a[5] = 30;
  {
    int[] a = int[10];
    a[5] = 14;
    println(a[5]);
  }
  return a[5];
}
println(a[5]);
println(f(int[10]));
