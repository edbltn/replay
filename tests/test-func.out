27
14
30
